meta-ros-common/conf/ros-distro/ros-distro.conf

```diff
@@ -40,7 +40,7 @@ ROS_DISTRO_METADATA_VERSION_REVISION = "${ROS_DISTRO_METADATA_MILESTONE_NUMBER}"
 ROS_DISTRO_METADATA_VERSION = "${ROS_DISTRO_METADATA_VERSION_MAJOR}.${ROS_DISTRO_METADATA_VERSION_MINOR}.${ROS_DISTRO_METADATA_VERSION_MICRO}.${ROS_DISTRO_METADATA_VERSION_UPDATE}"
 ROS_DISTRO_METADATA_VERSION .= "-${ROS_DISTRO_METADATA_VERSION_REVISION}"
 
-ROS_DISTRO ??= "${@d.getVar('ROS2_DISTRO') if d.getVar('ROS2_DISTRO') != '' else d.getVar('ROS1_DISTRO')}"
+ROS_DISTRO ??= "${@d.getVar('ROS2_DISTRO') if d.getVar('ROS2_DISTRO', True) not in (None, '') else d.getVar('ROS1_DISTRO', True)}"
```

Comment:
Isn't `True` the default the 2nd parameter of `d.getVar()`?

Reply:
Yes, I've added it when debugging why it didn't work and then forgot to drop it. Will do.